internal/src/clippy_utils.rs

```diff
@@ -4,6 +4,7 @@ use std::{
     fs::{read_to_string, write},
     path::Path,
 };
+use toml;
```

Comment:
```suggestion
```
Hmm. Here, too, a lint should have fired.

I hope you don't mind. I'm going to spend a day or two looking into this.

Reply:
This lint specifically should have fired: https://rust-lang.github.io/rust-clippy/master/index.html#/single_component_path_imports

Reply:
Leaving a note here for myself.

The reason for `single_component_path_imports` not firing seems related to `clippy_utils` being a public module: https://github.com/trailofbits/dylint/blob/6db435a98a7045fe64ce44a4c7901f56016c8c81/internal/src/lib.rs#L7-L8

If you remove the `pub`, re-add the `use toml;`, and run the following commands, a warning is generated:
```sh
cd internal
cargo clippy --all-features
```